CONTRIBUTING.md

```diff
@@ -42,5 +49,41 @@ Included in this repo is the file `helphub.wordpress.2017-06-15.xml`. Import thi
 6. Click the button to begin. It may take a while to complete. If there are some failed imports, it should still be okay.
 7. Your local install of HelpHub should now be ready to view in your browser.
 
+Once you have Helphub in your localhost up and running, you can contribute with pull requests either from your own fork or directly in this repository, after you're added to it as a contributor. We are using [Travis CL](https://travis-ci.org/) for tests on every pull request. You can, also, run these tests localy before pushing your code (more on this later). Development covers work on both, theme and plugins and requires following [best practices](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/) and [WordPress coding standards](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards).
```

Comment:
• `Once you have your local install of HelpHub up and running, you can contribute with pull requests either from your own fork or after you've added as a contributor directly in this repository.`

• Capitalize `[WordPress coding standards]`, `[WordPress Coding Standards]`